### DISCUSSION TITLE
Increase request timeout

### DISCUSSION THREAD

**Discussion by xykylikuf001** [ultrafunkamsterdam/undetected-chromedriver#1240] [Q&A]
Hello guys! Please help me.
Sometimes web service got a long response, which raise timeout error.
I need to increase the request timeout, but directly chrome disable this feature several years. 
It's possible do it on firefox like set the configuration network.http.response.timeout.
But I ask you it is possible to patch the chrome driver to increase timeout request?
Thanks!!!!

**Comment by damodharheadrun** [comment]
Hi @jba-tm 
Can you try the below solutions

` driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS); ----> JAVA
`

`driver.set_page_load_timeout(30) -- > Firefox, Chromedriver and undetected_chromedriver:
`

For more information please check the [link](https://stackoverflow.com/questions/17533024/how-to-set-selenium-python-webdriver-default-timeout)

**Comment by GoDesignpk** [comment]
Thanks for your help. But implicitlyWait and set_page_load_timeout are only the selenium waits. But I mean browsers requests timeout s. On chrome this is disabled several years (((. see this links [link](https://stackoverflow.com/questions/39751124/increase-timeout-limit-in-google-chrome#:~:text=Chrome%3A%20It%20is%20not%20possible,the%20timeout%20settings%20in%20Chrome.&text=IE%3A%20It's%20possible%20to%20change,timeout%20from%2060s%20to%2010min.).
I asking the it's possible to patch requests timeouts on chrome?
Thanks!!

**Reply by damodharheadrun** [reply]
ok @GoDesignpk ,

Thanks for  your information
